cri.go

```diff
@@ -82,6 +83,13 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 		return nil, errors.Wrap(err, "failed to create CRI service")
 	}
 
+	meta, err := ic.Get(plugin.MetadataPlugin)
```

Comment:
should this new block of code setting the contentStore variable be in the go func()?

Reply:
Nope. After all services become local services, we'll pass the client into `NewCRIContainerdService`, which is out of `go func()`.